tests/unit/libbpf_test.cpp

```diff
@@ -315,12 +315,10 @@ TEST_CASE("libbpf program", "[libbpf]")
 {
     _test_helper_libbpf test_helper;
 
-    struct bpf_object* object;
-    int program_fd;
-    int result = bpf_prog_load_deprecated("droppacket.o", BPF_PROG_TYPE_XDP, &object, &program_fd);
-    REQUIRE(result == 0);
+    struct bpf_object* object = bpf_object__open("droppacket.o");
```

Comment:
These changes are fine but we also need to add a test case with the tags "[libbpf][deprecated]" that explicitly still uses `bpf_prog_load_deprecated`

Reply:
I get `'bpf_prog_load_deprecated': Use bpf_object__open() and bpf_object__load() instead. [F:\ebpf-for-windows\my\tests\unit\test.vcxproj]` due to the way the function is annotated.

Reply:
Correct, you need to add the pragma like line 296 does for the same reason.  Every test in this file with the `[deprecated]` tag has a line like that.